lib/workos/user_management.rb

```diff
@@ -118,106 +118,63 @@ def authorization_url(
       end
       # rubocop:enable Metrics/ParameterLists
 
-      # Adds a User as a member of the given Organization.
+      # Gets a User
       #
       # @param [String] id The unique ID of the User.
-      # @param [String] organization_id Unique identifier of the Organization.
       #
       # @return WorkOS::User
       sig do
-        params(
-          id: String,
-          organization_id: String,
-        ).returns(WorkOS::User)
+        params(id: String).returns(WorkOS::User)
       end
-      def add_user_to_organization(id:, organization_id:)
+      def get_user(id:)
         response = execute_request(
-          request: post_request(
-            path: "/users/#{id}/organizations",
-            body: {
-              organization_id: organization_id,
-            },
+          request: get_request(
+            path: "/user_management/users/#{id}",
             auth: true,
           ),
         )
 
         WorkOS::User.new(response.body)
       end
 
-
-      # Deletes a User
+      # Retrieve a list of users.
       #
-      # @param [String] id The unique ID of the User.
+      # @param [Hash] options
+      # @option options [String] email Filter Users by their email.
+      # @option options [String] organization_id Filter Users by the organization
```

Comment:
nit:       `# @option options [String] organization_id Filter Users by the organizations they are members of`

Reply:
Fixed on https://github.com/workos/workos-ruby/pull/241/commits/892406b386619b4583e9e430d53e4f8a48cddc64